test/runtests.jl

```diff
@@ -194,6 +194,20 @@ end
     @test merge(h1,h2) == h1+h2
 end
 
+@testset "Rebinning" begin
+    h1 = Hist1D(rand(10^2), 0:0.1:1)
+    @test h1 == rebin(h1, 1)
+    @test integral(h1) == integral(rebin(h1, 5))
+    @test sum(h1.sumw2) == sum(rebin(h1, 5).sumw2)
```

Comment:
can we add some checks here to see if new bin edges meet our expectation?

Reply:
done